packages/runtime/test/integration/basic-teal.ts

```diff
@@ -12,76 +12,63 @@ import { expectRuntimeError } from "../helpers/runtime-errors";
 const minBalance = BigInt(ALGORAND_ACCOUNT_MIN_BALANCE + 1000); ; // 1000 to cover fee
 const initialJohnHolding = minBalance + 2000n;
 const initialBobHolding = minBalance + 500n;
+const fee = 1000;
 
 describe("Stateless Algorand Smart Contracts delegated signature mode", function () {
   useFixture("basic-teal");
   let john = new AccountStore(initialJohnHolding);
   let bob = new AccountStore(initialBobHolding);
+  const runtime = new Runtime([john, bob]);
 
-  // set up transaction paramenters
   const txnParams: ExecParams = {
     type: TransactionType.TransferAlgo, // payment
     sign: SignType.LogicSignature,
     fromAccountAddr: john.account.addr,
     toAccountAddr: bob.address,
     amountMicroAlgos: 100n,
-    lsig: {} as LogicSig, // populated below
-    payFlags: { totalFee: 1000 }
+    lsig: {} as LogicSig, // will be set below
+    payFlags: { totalFee: fee }
   };
 
-  let runtime: Runtime;
-  this.beforeAll(function () {
-    runtime = new Runtime([john, bob]); // setup test
-  });
-
-  // update account state after each execution
-  afterEach(function () {
+  // helper function
+  function syncAccounts (): void {
     john = runtime.getAccount(john.address);
     bob = runtime.getAccount(bob.address);
-  });
+  }
 
-  it("should send algo's from john to bob if stateless teal logic is correct", function () {
+  it("should send algo's from john to bob if delegated logic check passes", function () {
     // check initial balance
     assert.equal(john.balance(), initialJohnHolding);
     assert.equal(bob.balance(), initialBobHolding);
-    // get logic signature
+
+    // get delegated logic signature
```

Comment:
```suggestion
    // make delegated logic signature
```
I am not sure if "get" is the right word here. As we create (delegate) the logic signature at line 46.